### PR TITLE
fix: fixing styling issues after angular material upgrade

### PR DESCRIPTION
**Pull request by martin-trajanovski**
## Description

This PR aims to fix all styling and functionality issues left after angular material upgrade to latest.

## Motivation

The UI was broken after angular upgrade.

## Fixes:

* https://jira.esss.lu.se/browse/SWAP-3540
* https://jira.esss.lu.se/browse/SWAP-3551

## Changes:

https://github.com/SciCatProject/frontend/assets/6333063/af74f914-c4e6-4408-8307-085d39fb9de7

## Tests included/Docs Updated?

- [x] Included for each change/fix?
- [x] Passing? (Merge will not be approved unless this is checked) 
- [ ] Docs updated?
- [ ] New packages used/requires npm install? 
- [ ] Toggle added for new features?
- [ ] Requires update of SciCat backend API?

### CODE REVIEW

File: src/app/about/about/about.component.html [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
-          >
-        </li>
-      </ul>
+      <mat-card-content>
+        <span [innerHTML]="aboutText | linky"></span>
+        <br /><br />
+        <ul>
+          <li>
+            <a
+              *ngIf="facility === 'PSI'"
```
Comment (bpedersen2) [inline src/app/about/about/about.component.html:13]: Q: add this link unconditionally, not only for PSI? (could be a new PR)

File: src/app/about/about/about.component.html [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
-      <a *ngIf="facility === 'PSI'" href="{{ SNFLink }}" target="blank"
-        >Open Research Data.</a
-      >
-      <br /><br />
-      {{ termsTextContinued }}
-      <a *ngIf="facility === 'PSI'" href="{{ PSIDataPolicy }}" target="blank"
-        >Data Policy.</a
-      >
+      <mat-card-content>
+        {{ termsText }}
```
Comment (bpedersen2) [inline src/app/about/about/about.component.html:35]: Possibly needs better configuration options per facility ( new PR)

**Review by bpedersen2 — APPROVED** [review]
Look good to mre.

File: src/app/about/about/about.component.html (hunk 1 above)
Comment (nitrosx) [reply src/app/about/about/about.component.html:13]: Maybe it should be moved to configuration

File: src/app/about/about/about.component.html (hunk 2 above)
Comment (nitrosx) [reply src/app/about/about/about.component.html:35]: This can be implemented with "links" when we release them

**Review by nitrosx — APPROVED** [review]

**Pull request merged**